.github/meson-ios-arm64.ini

```diff
@@ -18,4 +18,4 @@ root = root
 [binaries]
 c = 'clang'
 strip = 'strip'
-pkgconfig = '/usr/local/bin/pkg-config'
+pkgconfig = 'pkg-config'
```

Comment:
`pkg-config = 'pkg-config'` while at it?

Reply:
Done in a separate patch on top. pkg-config is not supported by the version of meson provided by ubuntu 22.04. It needs to be bumped later on.